masterlist.yaml

```diff
@@ -17344,16 +17344,26 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Arena - An Encounter Zone Overhaul' ]
         condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
+    tag: [ C.Location ]
     clean:
+      # version: 1.1.1
       - crc: 0x683B8666
         util: 'SSEEdit v4.0.4'
+      # version: 1.1.2
```

Comment:
Version comments aren't really necessary as the version cleaned will be noted in the commit message. Also, cleaning data for old versions can just be replaced with new data:

```suggestion
```